ruby.spec

```diff
@@ -68,6 +68,9 @@ rm -rf $RPM_BUILD_ROOT
 
 %changelog
 
+* Thu Oct 19 2018 Masataka Suzuki <koshigoe@feedforce.jp> - 2.5.3
```

Comment:
It's `Fri` :)

Reply:
oops

Reply:
630c701d84c34e68dd6d43d40d961b52b6868f46
(🙇 amend & force pushed)